addons/website_sale/views/templates.xml

```diff
@@ -1221,7 +1221,6 @@
         <xpath expr="//div[@id='o_product_terms_and_share']" position="inside">
             <div class="h4 mt-3 mb-0 d-flex justify-content-md-end flex-shrink-0" contenteditable="false">
                 <t t-snippet-call="website.s_share">
-                    <t t-set="_exclude_share_links" t-value="['whatsapp', 'linkedin']"/>
```

Comment:
Why is _exclude_share_links completly deleted instead of being left empty for user to be able to exclude some share links?

Reply:
If someone adds the new share icon then again user has to add the `if` condition ([like this](https://github.com/odoo/odoo/pull/139674/files#diff-7e1212985d657cf2e9ffc9f338f4bd85f5a44b3b020dd4590ad66d25ff4e2512L7)). so, I completely removed it.

Reply:
I prefer to avoid changes in `website` if not necessary, there is no need to drop the support of `_exclude_share_links` variable.